react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1027 @@
+          </aside>
+        </section>
+
+        <section class="bullets-and-picture-slide">
+          <h4 class="slide-name">iOS</h4>
+          <div>
+            <img src="assets/images/ios_set.png" alt="simple" />
+            <ul>
+              <li>Xcode</li>
+              <li>Objective-C, Swift languages</li>
+              <li>Simulator</li>
+            </ul>
+          </div>
+          <aside class="notes">
+            iOS is the mobile operating system that runs on Apple`s mobile devices, most notably the iPhone and the iPad.<br>Native iOS applications can be written in <strong>Objective-C</strong> or <strong>Swift</strong>. Swift is a relatively newer language, introduced only in 2014, while Objective-C is more of an old school programming language.<br>Just like an APK (Android Application Package) file can be installed on Android devices, an IPA file can be used for <strong>testing iOS applications.</strong><br>An <strong>.ipa</strong> file is an iOS and iPadOS application archive file which stores an iOS/iPadOS app. Each .ipa file includes a binary and can only be installed on an iOS/iPadOS or ARM-based macOS device.<br>To develop iOS apps you should use Xcode. <strong>Xcode</strong> is the IDE (Integrated Development Environment) that Apple creates and uses for iOS and OS X development. <br>You can testing your app on an actual physical iOS device or the <strong>Simulator</strong>.<br>
+          </aside>
+        </section>
+
+        <section class="several-pictures-3-slide-with-description">
+          <h4 class="slide-name">Types of apps</h4>
+          <div>
+            <div class="fragment" data-fragment-index="1">
+              <img src="assets/images/native-type.png" alt="simple" />
+              <p>Native</p>
+            </div>
+            <div class="fragment" data-fragment-index="2">
```

Comment:
`data-fragment-index="3"` please set index for hybrid app to 3 and for web app to 2, so than images will be appear as they mention in speaker notes

Reply:
Done